src/packagedcode/plugin_package.py

```diff
@@ -272,7 +278,74 @@ def process_codebase(self, codebase, strip_root=False, package_only=False, **kwa
                 if TRACE_LICENSE and modified:
                     logger_debug(f'packagedcode: process_codebase: add_referenced_license_matches_from_package: modified: {modified}')
 
+@post_scan_impl
+class PackageSummary(PostScanPlugin):
+    """
+    Summary at the Package Level.
+    """
+    run_order = 11
+    sort_order= 11
+
+    options = [
+        PluggableCommandLineOption(('--package-summary',),
+        is_flag=True, default=False,
+        help='Summarize scans by providing License Clarity Score' 
+        'and populating other license/copyright attributes '
+        'for package instances.',
+        required_options=['classify'],
```

Comment:
Shouldn't `--package` also be a required option?

Reply:
Resolved